pygpu/gpuarray.pyx

```diff
@@ -1428,6 +1428,7 @@ cdef GpuArray pygpu_reshape(GpuArray a, unsigned int nd, const size_t *newdims,
             tot *= newdims[i]
     cdims = <size_t *>calloc(nd, sizeof(size_t))
     if cdims == NULL:
+        free(cdims)
```

Comment:
If cdims is NULL then it can't be allocated.  This is useless.

Reply:
Indeed.

Reply:
Fixed.